testdata/expected/docker/python-sample-0/python-sample-0/PipPackage.graph.json

```diff
@@ -5,429 +5,340 @@
       "UnitType": "PipPackage",
       "Unit": "python-sample-0",
       "Path": "pkg0",
-      "TreePath": "pkg0",
       "Name": "pkg0",
       "Kind": "module",
       "File": "pkg0/__init__.py",
       "DefStart": 0,
       "DefEnd": 0,
-      "Exported": true
+      "Exported": true,
+      "TreePath": "pkg0"
     },
     {
       "Repo": "github.com/sgtest/python-sample-0",
       "UnitType": "PipPackage",
       "Unit": "python-sample-0",
       "Path": "pkg0/m0",
-      "TreePath": "pkg0/m0",
       "Name": "m0",
       "Kind": "module",
       "File": "pkg0/m0.py",
       "DefStart": 0,
       "DefEnd": 0,
-      "Exported": true
+      "Exported": true,
+      "TreePath": "pkg0/m0"
     },
     {
       "Repo": "github.com/sgtest/python-sample-0",
       "UnitType": "PipPackage",
       "Unit": "python-sample-0",
       "Path": "pkg0/m0/Class0",
-      "TreePath": "pkg0/m0/Class0",
       "Name": "Class0",
       "Kind": "type",
       "File": "pkg0/m0.py",
       "DefStart": 60,
       "DefEnd": 66,
-      "Exported": true
+      "TreePath": "pkg0/m0/Class0"
     },
     {
       "Repo": "github.com/sgtest/python-sample-0",
       "UnitType": "PipPackage",
       "Unit": "python-sample-0",
       "Path": "pkg0/m0/Class0/meth0",
-      "TreePath": "pkg0/m0/Class0/meth0",
       "Name": "meth0",
       "Kind": "func",
       "File": "pkg0/m0.py",
       "DefStart": 124,
       "DefEnd": 129,
-      "Exported": true
-    },
-    {
-      "Repo": "github.com/sgtest/python-sample-0",
-      "UnitType": "PipPackage",
-      "Unit": "python-sample-0",
-      "Path": "pkg0/m0/Class0/meth0/self",
-      "TreePath": "pkg0/m0/Class0/meth0/self",
-      "Name": "self",
-      "Kind": "var",
-      "File": "pkg0/m0.py",
-      "DefStart": 130,
-      "DefEnd": 134,
-      "Exported": true
+      "TreePath": "pkg0/m0/Class0/meth0"
     },
     {
       "Repo": "github.com/sgtest/python-sample-0",
       "UnitType": "PipPackage",
       "Unit": "python-sample-0",
       "Path": "pkg0/m0/Class0/var0",
-      "TreePath": "pkg0/m0/Class0/var0",
       "Name": "var0",
       "Kind": "var",
       "File": "pkg0/m0.py",
       "DefStart": 107,
       "DefEnd": 111,
-      "Exported": true
```

Comment:
This will be fixed in a new branch which is based off this one just to keep the diff clean since it will also change program mode output.

The PR for the fix is: #26